data/mods/ssb/moves.ts

```diff
@@ -241,6 +241,28 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Dark",
 	},
 
+	// havi
+	augurofebrietas: {
+		accuracy: 100,
+		basePower: 70,
+		category: "Special",
+		shortDesc: "Disables the target's last move and pivots out.",
+		name: "Augur of Ebrietas",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit() {
+			this.attrLastMove('[anim] Spirit Shackle');
+		},
+		selfSwitch: true,
+		secondary: {
+			chance: 100,
+			volatileStatus: 'disable',
+		},
```

Comment:
```suggestion
		onHit(target, source, move) {
			target.addVolatile('disable', source);
		},
		secondary: null,
```

Reply:
did it like this because moves that inflict volatiles that are beneficial to the user are usually added as secondary effects (e.g. salt cure)

Reply:
I'd rather not use secondaries for effects unless the submitter says the move is a 100% chance (more like Sappy Seed, Thousand Waves, etc.), covert cloak and shield dust can cause unnecessary confusion.

Reply:
fair. will make it a regular `volatileStatus` then

Reply:
works for me